### ISSUE TITLE
Qwen2.5-VL-7B-Instruct 输出中断

### ISSUE THREAD

**Issue by winter-JX** [QwenLM/Qwen2.5-VL#1021]
使用Qwen2.5-VL-7B-Instruct 进行推理时，模型的输出总是戛然而止。
我的代码如下：
`
from transformers import Qwen2_5_VLForConditionalGeneration, AutoProcessor
from qwen_vl_utils import process_vision_info
import torch

model = Qwen2_5_VLForConditionalGeneration.from_pretrained(
    "/data/xinference_models/modelscope/models/Qwen/Qwen2.5-VL-7B-Instruct",
    torch_dtype=torch.bfloat16,
    attn_implementation="flash_attention_2",
    device_map="auto",
    token=32000
)

processor = AutoProcessor.from_pretrained("/data/xinference_models/modelscope/models/Qwen/Qwen2.5-VL-7B-Instruct")
messages= [
    {
        "role": "user",
        "content": [
            {
                "type": "text",
                "text": "Describe your understanding of operating systems"
            },
        ]
    }
]

# Preparation for inference
text = processor.apply_chat_template(
    messages, tokenize=False, add_generation_prompt=True
)
image_inputs, video_inputs = process_vision_info(messages)
inputs = processor(
    text=[text],
    images=image_inputs,
    videos=video_inputs,
    padding=True,
    return_tensors="pt",
)
inputs = inputs.to(model.device)

# Inference: Generation of the output
generated_ids = model.generate(**inputs, max_new_tokens=128)
generated_ids_trimmed = [
    out_ids[len(in_ids) :] for in_ids, out_ids in zip(inputs.input_ids, generated_ids)
]
output_text = processor.batch_decode(
    generated_ids_trimmed, skip_special_tokens=True, clean_up_tokenization_spaces=False
)
print(output_text)`

输出如下：['An operating system (OS) is the software that manages computer hardware and software resources and provides common services for computer programs. It is the foundation on which all other software, including application software, runs.\n\nThe primary functions of an operating system include:\n\n1. **Process Management**: Managing the execution of processes, which are instances of running programs. This includes scheduling processes to run in an efficient manner, handling multitasking, and managing memory allocation.\n\n2. **Memory Management**: Allocating and deallocating memory to processes as needed. This involves managing the physical memory (RAM) and virtual memory (swap space).\n\n3. **File System Management**:']

**可以看到输出停止在了File System Management，显然后面应该还应该有一段文字，这是什么原因，我该如何解决？**

**Comment by ShuaiBai623** [comment]
`generated_ids = model.generate(**inputs, max_new_tokens=128)`
the `max_new_tokens` is set 128

**Comment by winter-JX** [comment]
@ShuaiBai623 谢谢😂
